pkg/controller/plan/migrator/base/doc.go

```diff
@@ -58,6 +58,7 @@ var (
 			{Name: api.PhaseCreateInitialSnapshot},
 			{Name: api.PhaseWaitForInitialSnapshot},
 			{Name: api.PhaseStoreInitialSnapshotDeltas, All: VSphere},
+			{Name: api.PhaseVirtV2vInspection, All: RequiresConversion},
```

Comment:
@mansam I was trying to move this to a separate step, but the `PhaseCreateDataVolumes`, which is after this step, is still in the initilize step. So we would need to move it out, which didn't seem doable.
Got any ideas how we could have this as separate step. Didn't want the initialize step to take such long time.